rider/src/main/kotlin/com/jetbrains/rider/internal/dump/DumpUnityExplorerAction.kt

```diff
@@ -0,0 +1,35 @@
+package com.jetbrains.rider.internal.dump
```

Comment:
I hadn't noticed where this file was placed. Can we move it to `com.jetbrains.rider.plugins.unity.actions.internal` please?

Reply:
Sure, will do.